src/browsers/tob/shared/components/EffectGrid.js

```diff
@@ -31,19 +33,23 @@ const TableRow = styled.tr`
 `
 
 const TableCell = styled.td`
-  width: 350px;
-  min-width: 350px;
-  max-width: 350px;
+  width: 250px;
+  min-width: 250px;
+  max-width: 250px;
   height: 200px;
   text-align: left;
 `
 
 const EffectGrid = ({ columns, rows, data, missing, width, height, margin }) => {
   const _margin = useMargin(margin, { top: 4, bottom: 4 })
 
+  const validCellTypes = useMemo(() => {
```

Comment:
You have the right idea here, but now you've expanded the state management of excluded cell types into a another component that should be pure. State management is a complex problem, and in general it's better to keep this logic in as few places as possible to reduce app complexity and to prevent subtle bugs like state de-synchronization.  These changes should be reverted and lifted back to the parent page components where the `effectGridColumns` are computed, and here you can use a simple filter based on the `excludedColumns` memo you created previously.

Reply:
Great point - I've reverted these changes and added extra filtering to effectGrid and effectColumns

Reply:
Looks great!